app/code/Magento/Sitemap/Test/Unit/Model/SitemapTest.php

```diff
@@ -582,6 +582,10 @@ protected function getModelMock($mockBeforeSave = false)
             ->method('_getCurrentDateTime')
             ->willReturn('2012-12-21T00:00:00-08:00');
 
+        $model->expects($this->any())
+            ->method('_getBaseDir')
+            ->willReturn('');
+
```

Comment:
Hello @SilinMykola 
Please remove redundant operation `->expect($this->any())`

```suggestion
        $model->method('_getBaseDir')
            ->willReturn('');

```

Also, it can be removed from all code of this file

Reply:
Hello @andrewbess
Thanks for your review. I made changes, please check.